packages/api/cms-api/src/dam/files/dto/file.input.ts

```diff
@@ -89,6 +102,13 @@ export class CreateFileInput {
     // TODO is this validation even used?
     @IsObject()
     scope?: DamScopeInterface;
+
+    @IsUndefinable()
+    @ValidateIf((o) => o.importSourceType !== undefined)
```

Comment:
Please change the validator to be `@IsString()` and `@IsNotEmpty()` (remove the `@IsUndefinable()`, that is covered by `@ValidateIf()`. And please test if this validation works locally. Currently it doesn't do anything...

Reply:
And instead of `o` please use `input`